src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceStringArr = sourceString
+    .split(';')
+    .filter(str => str.includes(':') && str.trim().length > 1);
```

Comment:
The `&& str.trim().length > 1` part is unnessesary. Having ':' symbol, str.length will always be over 1 in any case, so this part of the condition will always be true and it just decreases readability. Generally, looks good to me. Fix this issue and everything is gonna be excellent )

Reply:
@dima8528 good point!